### DISCUSSION TITLE
Getting Started: error in total dose answer to Question 6.4.14?

### DISCUSSION THREAD

**Discussion by mjukdisk** [nrc-cnrc/EGSnrc#780] [Questions] [answered]
Asked this a while back on reddit, and if I recall I was asked to post it here instead... There's a question in the getting started manual, 6.4.14. You ask what is the total dose given this output:

```
==> Summary of region dosimetry (per particle)
ir medium rho/[g/cm3] V/cm3 Edep/[MeV] D/[Gy]
3 air 0.001 0.5773 9.4809e-09 +/- 1.125 % 2.1836e-15 +/- 1.125 %
7 air 0.001 0.0288 4.7904e-10 +/- 2.005 % 2.2117e-15 +/- 2.005 %
31 air 0.001 0.0592 1.0028e-09 +/- 1.921 % 2.2538e-15 +/- 1.921 %
```

The answer is given as the sum (6.6491x1E-15), but shouldn't it be the volume-averaged sum (2.1911x1E-15)?

**Comment by rtownson** [comment, marked as answer]
Thanks, this is indeed incorrect. I'll update it! The correct way is like this, using the energy deposited to the combination of regions:
```
(9.4809e-09 + 4.7904e-10 + 1.0028e-09) / (0.001*(0.5773 + 0.0288 + 0.0592))*1.602e-10
= 2.6397e-15
```

**Reply by rtownson** [reply]
@mjukdisk note that I made a mistake when I first replied, see the edited version above.

**Reply by mjukdisk** [reply]
Thanks!
I guess that's the pedagogical answer, however just by the values presented in the table, you'll get closer to the reported value "Geometry Cavity dose" by doing the volume average np.dot(Dose, volume)/(sum(volume)). I guess that's just a matter of significant values in the table.

While you're at it; I remember that the volumes on page 48 were in the wrong order when I did the tutorial, don't know if that was corrected or not.
volume = 0.02880 0.57732 0.05916
should be (?)
volume = 0.57732, 0.02880, 0.05916

**Reply by rtownson** [reply]
Good point, I'll include both answers in the solutions. Thanks for the other correction as well.
